pandas/tests/indexes/test_base.py

```diff
@@ -2614,3 +2614,22 @@ def construct(dtype):
         no_matches = np.array([-1] * 6, dtype=np.intp)
         tm.assert_numpy_array_equal(result[0], no_matches)
         tm.assert_numpy_array_equal(result[1], no_matches)
+
+
+@pytest.mark.parametrize(
+    "idx, target, expected",
+    [
+        ([np.nan, "var1", np.nan], [np.nan], np.array([0, 2], dtype=np.int64)),
```

Comment:
indexing with multiple nans is not very well supported at a high level, can you add a user facing example (is there on e in the OP)?

Reply:
What does OP stand for?

Reply:
Added the example from the OP in the `get_indexer_for` docs